dts/common/nordic/nrf54l15.dtsi

```diff
@@ -212,6 +212,7 @@
 				ieee802154-supported;
 				ble-2mbps-supported;
 				ble-coded-phy-supported;
+				cs-supported;
```

Comment:
Since there's a dedicated `bt_hci_controller` node (as a child node of the radio node), wouldn't these kind of properties be more suitable there? What's the rationale of putting Bluetooth-specific properties in the generic radio node instead of the Bluetooth-specific HCI device node?

Reply:
Hmm, just as I wrote that I realized that the upstream `zephyr,bt-hci-ll-sw-split` hasn't yet implemented channel sounding support. That said, the same question still applies, i.e. what's the reason you wouldn't want to put these into your HCI DT node for the SD controller?